week04/ecommerce/Index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link rel="stylesheet" media="screen and (max-width: 800px)" href="smallstyle.css">
+    <link rel="stylesheet" media="screen and (min-width: 800px)" href="fullstyle.css">
+</head>
+<header>
+    <!-- Navbar -->
+    <nav class="navbar">
+        <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="products.html">Products</a></li>
+            <li><a href="contact.html">Contact</a></li>
+        </ul>
+    </nav>
+    <!-- Search -->
+    <div class="search">
+        <label>Search:</label>
+        <input type="text">
+        <input type="submit">
+    </div>
```

Comment:
Try formatting this search box so that it doesn't push the rest of your page down. If you notice, the "Welcome" header and everything below it gets forced down.